torch/_dynamo/polyfill.py

```diff
@@ -45,3 +45,11 @@ def accumulate_grad(x, new_grad):
         x.grad = new_grad
     else:
         x.grad.add_(new_grad)
+
+
+def list_cmp(op, left, right):
```

Comment:
Mind adding typing that both are suppose to be SizedSequence. What if a user accidentally passed a tuple as one of the args?

Reply:
These functions aren't user-facing.